main.py

```diff
@@ -33,30 +44,32 @@ def setupSpotifyObject(username, scope, clientID, clientSecret, redirectURI):
     token = util.prompt_for_user_token(username, scope, clientID, clientSecret, redirectURI)
     return spotipy.Spotify(auth=token)
 
-def main(username, scope, clientID, clientSecret, redirectURI, path):    
+def main(username, scope, clientID, clientSecret, redirectURI, path):
     spotify = setupSpotifyObject(username, scope, clientID, clientSecret, redirectURI)
 
+    restartSpotify(path)
+
     while True:
-        
+
         try:
             current_track = spotify.current_user_playing_track()
-        except:
+        except Exception:
             print('token expired')
             spotify = setupSpotifyObject(username, scope, clientID, clientSecret, redirectURI)
             current_track = spotify.current_user_playing_track()
-            
+
         try:
             if current_track['currently_playing_type'] == 'ad':
                 restartSpotify(path)
                 print('Ad skipped')
         except TypeError:
             pass
-        
+
         time.sleep(1)
 
 if __name__ == '__main__':
-    
-    PATH = ''
+
+    PATH = ""

```

Comment:
See my comment on the PR. This should probably be set to None and then checked for is None in the code.